share/wake/lib/core/print.wake

```diff
@@ -38,75 +38,35 @@ from wake export type LogLevel
 export def getLogLevelName (LogLevel name): String =
     name
 
-# These colours are available for use with LogLevels
-export data Colour =
-    Black
-    Red
-    Green
-    Yellow
-    Blue
-    Magenta
-    Cyan
-    White
-
-export data Intensity =
-    Dim
-    Bright
-
-# makeLogLevel: define a new log level
-export def makeLogLevel (name: String) (colour: Option Colour): LogLevel =
-    makeLogLevel2 name colour None
-
-export def makeLogLevel2 (name: String) (colour: Option Colour) (intensity: Option Intensity): LogLevel =
-    def set name code = prim "colour"
-    def colourCode = match colour
-        None   = 0
-        Some c = 8 + match c
-            Black   = 0
-            Red     = 1
-            Green   = 2
-            Yellow  = 3
-            Blue    = 4
-            Magenta = 5
-            Cyan    = 6
-            White   = 7
-    def intensityCode = match intensity
-        None   = 0
-        Some i = 16 * match i
-            Dim    = 1
-            Bright = 2
```

Comment:
lol don't have to worry about formatting these anymore (do you need to rebase?)

Reply:
Yeah lol true haha. I probably need to rebase. You've been writing code like a mad man.